ga4gh/frontend.py

```diff
@@ -12,13 +12,31 @@
 import flask.ext.api as api
 import flask.ext.cors as cors
 
+from flask.ext.api.decorators import set_renderers
+from flask.ext.api.renderers import HTMLRenderer
+
 app = api.FlaskAPI(__name__)
 app.config.from_object('ga4gh.server.config:DefaultConfig')
 if os.environ.get('GA4GH_CONFIGURATION') is not None:
     app.config.from_envvar('GA4GH_CONFIGURATION')
 
+app.config['DEFAULT_RENDERERS'] = [
+    'flask.ext.api.renderers.JSONRenderer',
+    'flask.ext.api.renderers.HTMLRenderer',
+    'flask.ext.api.renderers.BrowsableAPIRenderer',
+]
```

Comment:
Looks like that's undergone some recent revision in Flask, and may not be
settled: https://github.com/tomchristie/flask-api/issues/8

I *think* the resolution is that they want [HTMLRenderer]

On Wed, Jan 28, 2015 at 10:41 PM, Steve Hershman <notifications@github.com>
wrote:

> In ga4gh/frontend.py
> <https://github.com/ga4gh/server/pull/103#discussion_r23744993>:
>
> >  app = api.FlaskAPI(__name__)
> >  app.config.from_object('ga4gh.server.config:DefaultConfig')
> >  if os.environ.get('GA4GH_CONFIGURATION') is not None:
> >      app.config.from_envvar('GA4GH_CONFIGURATION')
> >
> > +app.config['DEFAULT_RENDERERS'] = [
> > +    'flask.ext.api.renderers.JSONRenderer',
> > +    'flask.ext.api.renderers.HTMLRenderer',
> > +    'flask.ext.api.renderers.BrowsableAPIRenderer',
> > +]
>
> I needed these changes to get the homepage to render, but I fear this may
> break the BrowsableAPIRenderer. I tried using @set_renderers(HTMLRenderer)
> <http://www.flaskapi.org/api-guide/renderers/#htmlrenderer>, but am
> getting TypeError: 'type' object is not iterable
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ga4gh/server/pull/103/files#r23744993>.
>